jest.config.js

```diff
@@ -0,0 +1,18 @@
+const nextJest = require('next/jest');
+
+const createJestConfig = nextJest({
+  // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
+  dir: './',
```

Comment:
Do we want to place the test config file in the \_\_tests__ folder? I don't have a strong preference

Reply:
it's got to be at the project level according to jest docs